Pinta.Core/Managers/ChromeManager.cs

```diff
@@ -145,14 +145,23 @@ public void InitializeSimpleEffectDialog (SimpleEffectDialogHandler handler)
 		simple_effect_dialog_handler = handler;
 	}
 
-	public void ShowErrorDialog (Gtk.Window parent, string message, string body, string details)
-	{
-		error_dialog_handler (parent, message, body, details);
+	public async void ShowErrorDialog (
+		Gtk.Window parent,
+		string message,
+		string body,
+		string details)
+	{
+		string response = await error_dialog_handler (parent, message, body, details);
```

Comment:
Could the error dialog handler return a `DialogResponse` rather than `string`? It's not very clear what the string return value might contain

Reply:
I just made it return an `ErrorDialogResponse` :)